dev/ProjectReunion_BootstrapDLL/IDynamicDependencyLifetimeManager.h

```diff
@@ -0,0 +1,171 @@
+
+
+/* this ALWAYS GENERATED file contains the definitions for the interfaces */
```

Comment:
Why's this getting committed?

Reply:
It defines the interface that needs to be supported by the PackagedCOM OOP Server 'lifetime manager' in a per-app 'sidecar' helper package. Needed here as bootstrapping needs to CCI that object and use the interface to find the ProjectReunion Framework package so we can load Microsoft.ProjectReunion.dll for non-packaged processes